mypyc/genops.py

```diff
@@ -49,6 +49,21 @@ def type_to_rttype(typ: Type) -> RTType:
     assert False, '%s unsupported' % type(typ)
 
 
+class AssignmentTarget(object):
+    pass
+
+
+class AssignmentTargetRegister(AssignmentTarget):
+    def __init__(self, register: Register) -> None:
+        self.register = register
+
+
+class AssignmentTargetIndex(AssignmentTarget):
```

Comment:
I think that eventually we would just make this more general... We can decide when the time comes whether list should be a specific case